File: PROJECT/API/NVCMS.API.ReadGoogleSheet/Models/GoogleSheetRequest.cs
using System.ComponentModel.DataAnnotations;

namespace NVCMS.API.ReadGoogleSheet.Models
{
    public class GoogleSheetRequest
    {
        [Required]
        public string SpreadsheetId { get; set; } = string.Empty;
        
        public string Range { get; set; } = "Sheet1!A:D";
        public int eventCat_id { get; set; } 
    }
}

File: PROJECT/MODULES/NVCMS.Modules.Scheduler/Models/GoogleSheetRequest.vb
Imports System.ComponentModel.DataAnnotations
Imports DotNetNuke.UI.WebControls

Namespace NVCMS.API.ReadGoogleSheet.Models

    Public Class GoogleSheetRequest

        Public Property SpreadsheetId As String = String.Empty

        Public Property Range As String = String.Empty

        Public Property eventCat_id As Integer

    End Class

End Namespace
